src/Raven.Server/Documents/Queries/AST/StringQueryVisitor.cs

```diff
@@ -110,10 +110,15 @@ public override void VisitOrderBy(List<(QueryExpression Expression, OrderByField
             }
         }
 
-        public override void VisitDeclaredFunction(StringSegment name, string func, DeclaredFunction.FunctionType type)
+        public override void VisitDeclaredFunction(string func, DeclaredFunction.FunctionType type)
```

Comment:
why do we need the `type` here ? it seems like no one is using it

Reply:
Done